README.md

```diff
@@ -23,6 +23,7 @@ If you want to get involved with science and research software engineering, this
    - [Academic Labs](#academic-labs) 
    - [National Labs](#national-labs) 
  - [Psychology](#psychology)
+ - [Neuroscience](#neuroscience)
```

Comment:
Almost there! This link just needs to be moved up to be before Psychology.

Reply:
d'oh, should be fixed now

Reply:
Perfecto!